CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+## Your new git-extra command should support
+
+* OSX, Ubuntu, FreeBsd(You may need to browse their man page)
+* Bash 3.2+ (If you are not sure, [Bash versions](http://tldp.org/LDP/abs/html/bash2.html))
```

Comment:
Perhaps "Bash 3.2+ (if you are not sure, see [the Bash documentation](http://tldp.org/LDP/abs/html/bash2.html))"?